zipline/utils/cli.py

```diff
@@ -157,10 +161,43 @@ def run_pipeline(print_algo=True, **kwargs):
     end = pd.Timestamp(kwargs['end'], tz='UTC')
 
     symbols = kwargs['symbols'].split(',')
-
-    if kwargs['source'] == 'yahoo':
+    asset_identifier = kwargs['metadata_index']
+    if asset_identifier is None:
+        asset_identifier = 'symbol'
+
+    # Pull asset metadata
+    asset_metadata = kwargs.get('asset_metadata', None)
```

Comment:
The first one, asset_identifier, is only used if a metadata file is provided. I've refactored to set 'symbol' as the arg default, rather than do this none-checking.